src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -354,6 +356,10 @@ class OPENASSETIO_CORE_EXPORT Manager final {
    *  distributed between processes to associate subsequent API usage
    *  with the supplied context.
    *
+   *  @note This should not be used within the same process to store
+   *  a context for use between subsequent API calls or threads. Retain
+   *  the Context object directly in this situation.
+   *
```

Comment:
[Consider] Feeling here is that the prior framing prescriptively implies it is forbidden and there's a special reason things will go wrong if you do, (which I don't _think_ is the case,) whereas this framing does a better job at informing the user why they wouldn't want to use it.
```suggestion
   *  @note Using this within the same process to store a context
   *  for use between subsequent API calls or threads is redundant
   *  Retain the Context object directly in this situation.
   *
```

Reply:
_Much_ better, thanks. Updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/1120/commits/3616d47df4d640590c7cc75dd9a90dbeed7404e4).